modules/concertBidAdapter.js

```diff
@@ -49,6 +52,8 @@ export const spec = {
     };
 
     payload.slots = validBidRequests.map(bidRequest => {
+      collectEid(eids, bidRequest);
```

Comment:
Should the object returned from this function call be added to the eids array declared above?